students/byoungju/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    name     = models.CharField(max_length=45)
+    email    = models.CharField(max_length=100)
+    password = models.CharField(max_length=45)
+    phone    = models.CharField(max_length=20)
+    birth    = models.DateField()
+
+    class Meta:
+        db_table = "users"
```

Comment:
```suggestion
class User(models.Model):
    name     = models.CharField(max_length=45)
    email    = models.CharField(max_length=200)
    password = models.CharField(max_length=45)
    phone    = models.CharField(max_length=45)
    birth_date = models.DateField()

    class Meta:
        db_table = "users"
```
병주님 변수명, 컨벤션 너무 좋습니다! 다만 변수명은 명확할수록 좋고, max_length 값도 여유있게 지정해주시는 게 좋습니다!

Reply:
변수명, max_length 값 변경했습니다!